overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -11463,7 +11463,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§6Sodium Hydroxide§r is an ingredient in §9Bio Diesel§r.\n\nThe primary sources are:\n\n- Combining §9Water§r and §6Sodium§r in a §3Chemical Reactor§r (also a good source of §9Hydrogen§r)\n- Splitting §9Salt Water§r in an §3Electrolyzer§r (also the best source of §9Chlorine§r).",
+          "desc:8": "§6Sodium Hydroxide§r is an ingredient in §9Bio Diesel§r.\n\nThe primary sources are:\n\n- Combining §9Water§r and §6Sodium§r in a §3Chemical Reactor§r (also a good source of §9Hydrogen§r)\n- Splitting §9Salt Water§r in an §3Electrolyzer§r (also a good source of §9Chlorine§r).\n- Combining §6Sodium Bisulfate§r and §9Water§r in a §3Chemical Reactor§r (The production of §6Sodium Bisulfate§r also yields a little §9Hydrochloric Acid§r)",
```

Comment:
I think the second highlighting of Sodium Bisulfate here is unnecessary but otherwise the quest looks good.

Reply:
I added the information after the Sodium Bisulfate because the other entries had additional information added to them about secondary products, and also Hydrochloric acid is an important substance.